01/PrintRecipe.cpp

```diff
@@ -6,15 +6,15 @@ using namespace std;
 
 namespace samples
 {
-	void PrintRecipe()
+	void PrintRecipeExample()
 	{
-		float breakfastPrice = 15.104f;
 		float icedcoffeePrice = 1.25f;
 		float lattePrice = 4.75f;
+		float breakfastPrice = 15.104f;
 
-		cout << "**" << setw(16) << setfill('=') << "**" << setw(8) << setfill('=') << "**" << setfill(' ') << endl;
+		cout << "--------------------------" << endl;
 		cout << setw(6) << "Name" << setw(17) << "Price" << endl;
-		cout << "**" << setw(16) << setfill('=') << "**" << setw(8) << setfill('=') << "**" << setfill(' ') << endl;
+		cout << "--------------------------" << endl;
 
 		cout << setw(13) << "Iced coffee" << setw(6) << "$" << "0" << icedcoffeePrice << endl;
 		cout << setw(7) << "Latte" << setw(12) << "$" << "0" << lattePrice << endl;
```

Comment:
we have these these:

setw 6, 17
setw 13 6
setw 7 12

is there any way we can use same setw for each line (I think that makes much more sense if we use for loop on the future

so something like this 

setw(10) << "Name << set(8) << price      // but both name and price is center aligned.. or name becomes left aligned.. while price becomes right aligned?

setw(10) << "lattel" << set(8) << price